test/integration/targets/junos_acls/tests/netconf/merged.yaml

```diff
@@ -0,0 +1,86 @@
+---
+- debug:
+    msg: "START junos_acls merged integration tests on connection={{ ansible_connection }}"
+
+- block:
+  - include_tasks: _reset_config.yaml
+
+  - set_fact:
+      config:
+        - acls:
+            - aces:
+                - name: ten_dot
+                  source:
+                    address: 10.0.0.0/8
+                    port_protocol:
+                      eq: bgp
+                  protocol: tcp
+                - name: eleven_dot
+                  source:
+                    address: 11.0.0.0/8
+                    port_protocol:
+                      eq: bgp
+                  protocol: tcp
+              name: initial_acl
+            - aces:
+                - name: twelve_dot
+                  source:
+                    address: 12.0.0.0/8
+                    port_protocol:
+                      eq: bgp
+                  protocol: tcp
+              name: second_acl
+            - aces:
+                - name: ssh_rule
+                  source:
+                    port_protocol:
+                      eq: 22
+                  protocol: tcp
+              name: allow_ssh_rule
```

Comment:
this should be allow_ssh_acl, otherwise it won't match to result.after.

Reply:
Thanks for looking all these up for me! I figured there was probably a transcription error in there.